src/main/resources/static/events-feedback.js

```diff
@@ -38,9 +38,9 @@ function feedbackNext() {
     $.ajax({
         url: baseApp + "/submission/update",
         type: "POST",
-        data: {step: "overview"},
+        data: {step: "odm-upload"},
         success: function () {
-            window.location.href = baseApp + "/views/final";
+            window.location.href = baseApp + "/views/odm-upload";
```

Comment:
Hi Bo,
the function 'provide_filled_template_upload' adds a HTML-form to the page using DHTML javascript. This form contains a Radio-Button with which the user can specify the CRF-status after the upload. We need this option because it costs us a lot of work/time if we have to change the CRF-status later on.